src/Sylius/Bundle/OrderBundle/Command/RemoveExpiredCartsCommand.php

```diff
@@ -27,7 +27,7 @@ protected function configure()
     {
         $this
             ->setName('sylius:remove-expired-carts')
-            ->setDescription('Removes carts that have been idle for a configured period. Configuration parameter - sylius_order.cart_expires_after.');
+            ->setDescription('Removes carts that have been idle for a configured period. Configuration parameter - sylius_order.cart_expiration_period.');
```

Comment:
@sweoggy in the end, only the bundle configuration would be the one under BC promise